src/js/utils.js

```diff
@@ -111,6 +111,19 @@ function getBounds(el) {
     box[k] = rect[k];
   }
 
+  // Take into account DOMElements that reside in another Frame by adjusting the
+  // getBoundingClientRect properties by the outer Frame's getBoundingClientRect offset
+  if (doc !== document) {
+    let frameElement = doc.defaultView.frameElement;
+    if (frameElement) {
+      let frameRect = getBounds(frameElement);
+      box.top += frameRect.top;
+      box.bottom += frameRect.top;
```

Comment:
`top` is added to `bottom` here because `height = top - bottom`, so we need to adjust `top` and `bottom`  the same offset amount